server/controllers/Contacts.js

```diff
@@ -5,7 +5,7 @@ const User = db.User;
 
 // Create and Save a new Contact
 module.exports.addContact = async (req, res) => {
-    let userId = req.params.userId;
+    let userId = req.user.id;
```

Comment:
Comment below can be removed because we no longer need to add that check since the user id comes from the jwt